radixconfig_playground.yaml

```diff
@@ -33,6 +37,7 @@ spec:
             ONESEISMIC_API_TRUSTED_PROXIES: ""
             ONESEISMIC_API_BLOCKED_IPS: ""
             ONESEISMIC_API_BLOCKED_USER_AGENTS: ""
+            OPENVDS_AZURE_IOMANAGER_THREAD_COUNT: ""
```

Comment:
Is this change ARM specific?

Reply:
No, it is not.

I look at playground config like something third-priority now. It is useful sometimes, most of the time it isn't.

So last time I updated the variables I for some reason didn't update playground file. With this new change it was of interest, as I wanted to test arm performance and you can't do that without changing the config and config is the same for all environments.
So I decided to add the missing parts, but didn't want to create a separate commit. Probably should have had.